pkgs/by-name/yc/ycm-cmake-modules/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ycm-cmake-modules";
+  version = "0.18.3";
+
+  src = fetchFromGitHub {
+    owner = "robotology";
+    repo = "ycm-cmake-modules";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-a4AC3TOjv50cVT0ARM1KFx2Z4ETzR5eDy761tnqkm8A=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  meta = {
+    description = "Collection of various useful CMake modules";
```

```suggestion
    description = "Collection of CMake modules for build maintenance";
```

Drops subjective/vague language in description.